src/NUnitFramework/framework/Internal/Filters/NotFilter.cs

```diff
@@ -57,15 +57,15 @@ public NotFilter( TestFilter baseFilter)
         /// <returns>True if the test passes the filter, otherwise false</returns>
         public override bool Pass(ITest test)
         {
-            return !BaseFilter.Pass(test);
-        }
+			return !BaseFilter.Match (test) && !BaseFilter.MatchParent (test);
+		}
 
-        /// <summary>
-        /// Check whether the filter matches a test
-        /// </summary>
-        /// <param name="test">The test to be matched</param>
-        /// <returns>True if it matches, otherwise false</returns>
-        public override bool Match( ITest test )
+		/// <summary>
+		/// Check whether the filter matches a test
+		/// </summary>
+		/// <param name="test">The test to be matched</param>
+		/// <returns>True if it matches, otherwise false</returns>
+		public override bool Match( ITest test )
```

Comment:
See EditorConfig.org.

It requires a VS plugin. I don't use it so I had forgotten it was there.

Our policy is that it's up to the developer to follow the standard but we
don't dictate what tools to use. We will add a control file for any
reasonable tool, but all are only there to use if you want them.

Remember, even using VS is a developer choice and some don't use it.

Maybe I should post this somewhere!
On Oct 4, 2015 3:30 AM, "Neil Colvin" <notifications@github.com> wrote:

> In src/NUnitFramework/framework/Internal/Filters/NotFilter.cs
> <https://github.com/nunit/nunit/pull/862#discussion_r41096552>:
>
> >
> > -        /// <summary>
> > -        /// Check whether the filter matches a test
> > -        /// </summary>
> > -        /// <param name="test">The test to be matched</param>
> > -        /// <returns>True if it matches, otherwise false</returns>
> > -        public override bool Match( ITest test )
> > +		/// <summary>
> > +		/// Check whether the filter matches a test
> > +		/// </summary>
> > +		/// <param name="test">The test to be matched</param>
> > +		/// <returns>True if it matches, otherwise false</returns>
> > +		public override bool Match( ITest test )
>
> There is an .editorconfig file in the solution. However, it seems to do
> nothing in VS 2015.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/nunit/nunit/pull/862/files#r41096552>.
>